### ISSUE TITLE
Accessibility improvements and issues 

### ISSUE THREAD

**Issue by simonssspirit** [telerik/kendo-react#473]
- failed purpose of a link must be described by its link text alone, or by the link text together with preceding page context -  This is in the Grid header cells and Pager

- Keyboard interaction:8 FailedA custom widget must support the keyboard interaction specified by its design - Error is shown in the DropDown, PanelBar, Toolbar, and Calendar.

- Expected input:8 FailedA custom widget must have a label and/or instructions that identify the expected input - Error is shown in the DropDown, PanelBar, Toolbar, and Calendar.

- aria-input-field-name - The DropDownList requires a title or aria-label attribute. The current workaround is to add the title programmatically:
https://stackblitz.com/edit/react-iwzah9?file=app/main.jsx

- Label missing in our filterable Grid inputs: I reproduced this in our filterable demo. Indeed we are adding the inputs there:
https://stackblitz.com/edit/react-jcaavc?file=app/main.jsx

- The components are missing the aria-valid tag:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-invalid_attribute

**Comment by kdikov82** [comment]
The Grid Pager buttons should include aria-label for screen readers as well (like "Previous page", "Next page", "Page 1", "Page 2", etc.)
